samples/wrappture/Makefile

```diff
@@ -2,37 +2,57 @@
 
 # Change the following to match your installation
 BOINC_DIR = ../..
-BOINC_API_DIR = $(BOINC_DIR)/api
-BOINC_LIB_DIR = $(BOINC_DIR)/lib
-RAPPTURE_DIR = /home/boincadm/rappture/rappture/src/core
-
-CXXFLAGS = -g \
+BOINC_SOURCE_API_DIR = $(BOINC_DIR)/api
+BOINC_SOURCE_LIB_DIR = $(BOINC_DIR)/lib
+BOINC_SOURCE_ZIP_DIR = $(BOINC_DIR)/zip
+
+BUILD_DIR = $(BOINC_DIR)/3rdParty/buildCache
+
+ifndef ANDROID
+  BOINC_API_DIR = $(BOINC_SOURCE_API_DIR)
+  BOINC_LIB_DIR = $(BOINC_SOURCE_LIB_DIR)
+  BOINC_ZIP_DIR = $(BOINC_SOURCE_ZIP_DIR)
+  VCPKG_DIR     = $(BUILD_DIR)/vcpkg/installed/x64-linux
+  LINUX_LDFLAGS = libstdc++.a -lpthread
+  LINUX_STDLIB  = libstdc++.a
+else
+  BOINC_API_DIR = $(TCINCLUDES)/lib
+  BOINC_LIB_DIR = $(TCINCLUDES)/lib
+  BOINC_ZIP_DIR = $(TCINCLUDES)/lib
+  LINUX_LDFLAGS = 
+  LINUX_STDLIB  = 
+endif
+
+RAPPTURE_DIR = $(VCPKG_DIR)/include/rappture
```

Comment:
If user has already `RAPPTURE_DIR` defined, it will be overwritten here, and this is not good

Reply:
In the original script, it not define by user, it define in the makefile.

Reply:
So let's make it user configurable ;)

Reply:
ok :)

Reply:
I change